lib/rails_event_store/client.rb

```diff
@@ -1,7 +1,7 @@
 module RailsEventStore
   class Client
 
-    def initialize(repository: RailsEventStoreActiveRecord::EventRepository.new,
```

Comment:
The idea of RES gem (and a separation a RubyEventStore gem) was to have an "umbrella" gem, one gem to install to have complete setup, with defaults. This change will break that assumption. You will have to remember to add event repository gem in the Gemfile. You will have to remember to setup proper event repository.